Src/EmbeddedScripts.JS.Jint/JsCodeRunnerOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using EmbeddedScripts.Shared;
+
+namespace EmbeddedScripts.JS.Jint
+{
+    public class JsCodeRunnerOptions
+    {
+        public JsCodeRunnerOptions Register<T>(T obj, string alias)
+        {
+            Container.Register(obj, alias);
+            return this;
+        }
+
+        public static JsCodeRunnerOptions Default => new();
```

Comment:
Why to create an empty instance? Passing nulls for default options should work just fine